yarn-project/sequencer-client/src/tx_validator/gas_validator.ts

```diff
@@ -47,7 +47,10 @@ export class GasTxValidator implements TxValidator<Tx> {
       maxFeesPerGas.feePerL2Gas.lt(this.#gasFees.feePerL2Gas);
 
     if (notEnoughMaxFees) {
-      this.#log.warn(`Skipping transaction ${tx.getTxHash()} due to insufficient fee per gas`);
+      this.#log.warn(`Skipping transaction ${await tx.getTxHash()} due to insufficient fee per gas`, {
```

Comment:
I suppose the alternative to this making the whole func async would be to kick this off as an async side-effect. But perhaps would get weird

Reply:
The caller was already async, so I didn't worry too much